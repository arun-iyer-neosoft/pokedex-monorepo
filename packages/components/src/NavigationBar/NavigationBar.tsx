import React from 'react';

interface NavigationBarProps {
	Brand: React.ReactNode;
}

export const NavigationBar = ({ Brand }: NavigationBarProps) => {
	return (
		<nav className='bg-blue-600 p-6'>
			<div className='max-w-7xl mx-auto flex items-center justify-between'>
				<div>{Brand}</div>
				<div></div>
			</div>
		</nav>
	);
};
