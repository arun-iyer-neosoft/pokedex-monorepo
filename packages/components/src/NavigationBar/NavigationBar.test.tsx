import { render, screen } from '@testing-library/react';
import * as React from 'react';
import { NavigationBar } from './NavigationBar';

describe('NavigationBar', () => {
	it('Renders the Navigation Bar with brand text', () => {
		const { container } = render(<NavigationBar Brand='Pokedex' />);

		expect(screen.getByText('Pokedex')).toBeInTheDocument();
		expect(container).toMatchSnapshot();
	});
});
