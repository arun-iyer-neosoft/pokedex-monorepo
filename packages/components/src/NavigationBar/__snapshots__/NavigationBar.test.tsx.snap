// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar Renders the Navigation Bar with brand text 1`] = `
<div>
  <nav
    class="bg-blue-600 p-6"
  >
    <div
      class="max-w-7xl mx-auto flex items-center justify-between"
    >
      <div>
        Pokedex
      </div>
      <div />
    </div>
  </nav>
</div>
`;
